curriculum/challenges/english/25-front-end-development/workshop-musical-instrument-inventory/6895e3f91c069e0068a42582.md

```diff
@@ -0,0 +1,42 @@
+---
+id: 6895e3f91c069e0068a42582
+title: Step 9
+challengeType: 20
+dashedName: step-9
+---
+
+# --description--
+
+Inside your `play` method, use `print()` to display a message about playing the instrument.
+
+Remove the `pass` keyword and use an f-string to include the instrument's name in the message: `"The [instrument name] is fun to play!"`. The `[instrument name]` should be replaced with the actual name of the instrument.
+
+# --hints--
+
+Your `play` method should use an f-string to display the message `"The {self.name} is fun to play!"` with the instrument name.
```

Comment:
It's confusing to use quotes and parenthesis in different ways. If the description uses square brackets as placeholders, we should either do the same in the hint or specify the actual content to print.

```suggestion
Your `play` method print `f"The {self.name} is fun to play!"`.
```

Reply:
I didn't understand this, is there something missing in the suggestion?